pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -635,6 +641,9 @@ public void setOfflineSegmentIntervalCheckerInitialDelayInSeconds(long initialDe
   }
 
   public long getPeriodicTaskInitialDelayInSeconds() {
+    if (containsKey(ControllerPeriodicTasksConf.SEGMENT_LEVEL_VALIDATION_INTERVAL_IN_SECONDS)) {
```

Comment:
This one is the generic PeriodicTask initial delay. The segment level validation seconds shouldn't be applied here. 
Maybe we need another on getRealtimeSegmentValidationInitialDelay()

Reply:
No. The code calls this method  to get the initial delay